src/components/navigation/header/NavigationItems.tsx

```diff
@@ -53,17 +54,19 @@ export const NavigationItems: FC<NavigationItemsProps> = ({
           alt="Platform logo"
         />
       </Link>
-      <NavigationItem
-        translationKey="header.search"
-        drawerHandler={searchDrawerHandler}
-        isOpen={isOpen && drawer?.current === DrawerNode.Search}
-      />
+      <Hide below="xs">
```

Comment:
i wonder if it's really worth optimizing for this small user group and implement a hamburger menu just for this little audience. Or is it planned to expand this further?